rcl/src/rcl/logging.c

```diff
@@ -77,6 +81,7 @@ rcl_logging_configure(const rcl_arguments_t * global_args, const rcl_allocator_t
   g_rcl_logging_ext_lib_enabled = !global_args->impl->log_ext_lib_disabled;
   rcl_ret_t status = RCL_RET_OK;
   g_rcl_logging_num_out_handlers = 0;
+  (void) rcutils_atomic_fetch_add_uint64_t(&__rcl_logging_init_count, 1);
```

Comment:
if multiple contexts exist, this will be called more than once. so that there should be no warning.